int32 drone_id

float32 local_complexity